launchable/commands/subset.py

```diff
@@ -254,9 +254,10 @@ def default_path_builder(file_name):
                     if path:
                         self.test_paths.append(self.to_test_path(path))
 
-        def get_payload(self, session_id, target, duration):
+        def get_payload(self, session_id, target, duration, test_runner):
             payload = {
                 "testPaths": self.test_paths,
+                "testRunner": test_runner,
```

Comment:
The field name reflects our public document https://docs.launchableinc.com/resources/integrations.

Reply:
Could you add this field to record tests command too?

Reply:
Sure! Seems it's redundant but is this correct place if we think of the current payload structure?

```json
{
  "events": [
    {
      "type": "case",
      "testPath": [
        {
          "type": "file",
          "name": "components/layouts/loading/snapshot.test.tsx"
        },
        { "type": "testcase", "name": "Renders correctly" }
      ],
      "testRunner": "jest",   // this guy!
      "duration": 0.047,
      "status": 1,
      "stdout": "",
      "stderr": "",
      "created_at": "2021-11-09T05:36:19",
      "data": null
    },
```

https://github.com/launchableinc/cli/blob/3baf8d6068d6fe039064a9559e090fffb219d3b8/tests/data/jest/record_test_result.json#L3-L18

Reply:
In my image, like this
```
{
  "events": [],
   "testRunner": "jest"   // this guy!
}
```

What do you think?

Reply:
OK, that's simpler!

Reply:
added with https://github.com/launchableinc/cli/pull/451/commits/9f901a3be1513dd0cb23068ccf63260704b747d4